src/scope/scope.js

```diff
@@ -158,12 +149,12 @@ export default class Scope {
    * @param {BitId} id
    */
   static getComponentRelativePath(id: BitId, scopePath?: string): string {
-    const relativePath = pathLib.join(id.name, id.scope);
+    const relativePath = pathLib.join(id.name, id.scope || '');
```

Comment:
Previously, in case id.scope was null, `pathLib` would throw an error. (not ideal of course, the error would be obscure to the end user).
With this change, it'll suppress the error and return an incorrect path.

What should have been done here is making sure that `id.scope` and `id.version` are defined, and throwing an error if they don't.

Reply:
Will be fixed.